### DISCUSSION TITLE
Icons in Heading (Popup Card) missing

### DISCUSSION THREAD

**Discussion by darki79** [Clooos/Bubble-Card#1042] [Questions about config, custom styles and templates] [answered]
Moin,

since the last update today die Icons in my Headings are missing:

<img width="604" alt="image" src="https://github.com/user-attachments/assets/4196f2b2-2f2a-44a3-b546-3414a4280a89" />

Do I have to adapt the yaml?

```
 - type: vertical-stack
    cards:
      - type: custom:bubble-card
        card_type: pop-up
        hash: "#raum-wohnzimmer"
        show_header: true
        name: " "
        icon: fas:couch
        styles: |-
          .bubble-icon {
            color: #926BC7 !important;
          }
          .bubble-sub-button-1 {
            background-color: #482422 !important;
          }
          .bubble-sub-button-2 {
            background-color: #213547 !important;
          }
          .bubble-button-background {
            opacity: 0.2 !important;
            background-color: #926BC7 !important;
          }
        bg_blur: "20"
        card_layout: large
        button_type: state
        sub_button:
          - entity: sensor.temp_wozi_average
            show_attribute: false
            show_state: true
            show_background: true
          - entity: sensor.temp_wozi_tisch_humidity
            show_attribute: false
            icon: mdi:water
            show_state: true
            show_background: true
```

Best,
Marcel

**Comment by MrBearPresident** [comment, marked as answer]
Hi

Due to some issues identified in the latest releases (v2.3.1, v2.3.2, v2.3.3), I have decided to mark them as pre-release versions. These updates may not be fully stable and could cause unexpected behavior. I strongly recommend rolling back to the latest stable release, **v2.3.0**, until further notice. This version has been thoroughly tested and provides a reliable experience. Thank you for your understanding and patience as we work to ensure a future updates that meets the highest standards of quality.

---

[u]**How to rollback**[/u]
1. Click on the integration  in HACS.
2. Click on the three dots icon top right of the page.
3. Select Redownload.
4. Choose the version you want from the dropdown list.
5. Click Download.
